qabelbox/src/main/java/de/qabel/qabelbox/fragments/CreateIdentityHeaderFragment.kt

```diff
@@ -0,0 +1,48 @@
+package de.qabel.qabelbox.fragments
+
+import android.app.Fragment
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import de.qabel.qabelbox.R
+import de.qabel.qabelbox.ui.extensions.setOrGone
+import kotlinx.android.synthetic.main.fragment_create_identity_header.*
+
+class CreateIdentityHeaderFragment : Fragment() {
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        return inflater.inflate(R.layout.fragment_create_identity_header, container, false)
+    }
+
+    override fun onViewCreated(view: View?, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        updateUI(null)
+    }
+
+    fun updateUI(name: String?, secondLine: String = "", thirdLine: String = "") {
+        if (name == null || name.isEmpty()) {
```

Comment:
It was a guardclause before and it really looks like a precondition. It's easier to read code like
```KOTLIN
if (!precondition) {
    // do minimal stuff to preserve a clean state
    return
}

// do the real logic now
...
```
then to understand that the if-branch it that precondition check and failure handling and that the else branch is what `updateUI` is ment to do on it's full extend. So I would prefer the structure this method had in java. Guard clauses can easy reading code a lot if you get used to it and especially in java it is very common (and I would expect the same from kotlin).